packages/js/test/plugins/auctionHouseModule/buy.test.ts

```diff
@@ -0,0 +1,68 @@
+import test, { Test } from 'tape';
+import spok, { Specifications } from 'spok';
+import { sol, token } from '@/types';
+import {
+  createNft,
+  createWallet,
+  killStuckProcess,
+  metaplex,
+  spokSameAmount,
+  spokSamePubkey,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { findAssociatedTokenAccountPda, Purchase } from '@/index';
+
+killStuckProcess();
+
+test('[auctionHouseModule] buy on an Auction House', async (t: Test) => {
```

Comment:
We need more tests (for both selling and buying):
- One with minimum input: allowing all default values to do their job and asserting it works.
- One with maximum input: making sure the operation still works when we provide explicit values for everything.
- One that asserts the operation fails is the input is incorrect (e.g. wrong price).
- When buying, one that checks partial buys as soon as the other PR is merged.